UNRELEASED.md

```diff
@@ -37,6 +37,8 @@ Use [the changelog guidelines](/documentation/Versioning%20and%20changelog.md) t
 - Remove filter functions ([#4650](https://github.com/Shopify/polaris-react/pull/4650))
 - Remove all color() invocations ([#4636](https://github.com/Shopify/polaris-react/pull/4636))
 - Cleaned up Button styling and $button-filled mixin([#4635](https://github.com/Shopify/polaris-react/pull/4635))
+- Removed miscellaneous css custom properties ([#4620](https://github.com/Shopify/polaris-react/pull/4620))
+- Removed remaining color sass function uses ([#4655](https://github.com/Shopify/polaris-react/pull/4655))
```

Comment:
```suggestion
```

Lets remove these as it hasn't released. Lets also add a `skip changelog` label so the test passes.